### PR TITLE
[fix] Sprache & CodeCharts Test Trial

### LINKED ISSUE #13: Einheitliche Sprache für Variablen, Dokumentation und GUI
- [x] Variablen (Englisch)
- [x] Dokumentation (Deutsch)
- [x] GUI (Deutsch)

### PR DESCRIPTION
**Pull request by ghost**
closes #13

### CODE REVIEW

File: src/main/java/github/weichware10/toolbox/codecharts/CodeChartsCoordinator.java [hunk 1]
```diff
@@ -60,7 +60,7 @@ public CodeChartsCoordinator(ConfigClient configClient, DataBaseClient dataBaseC
         this.imageView = imageView;
         this.codeCharts = codeCharts;
         codeChartsInput = new CodeChartsInput();
-        // BILD SETZEN
+        // Bild setzten
```
Comment (joshuajeschek) [inline src/main/java/github/weichware10/toolbox/codecharts/CodeChartsCoordinator.java:63]:
```suggestion
        // Bild setzen
```

File: src/main/java/github/weichware10/toolbox/gui/Tutorial.java [hunk 2]
```diff
@@ -71,7 +71,7 @@ public Tutorial(Stage primaryStage, ConfigClient configClient, DataBaseClient da
         primaryStage.heightProperty().addListener(
                 o -> controller.getImageView().setFitHeight(primaryStage.getHeight() / 1.2));
 
-        //Kümmert sich um das Laden aller Bilder und setzt das erste Bild
+        //Zuständig für das Laden aller Bilder und Setzten des ersten Bilds
```
Comment (joshuajeschek) [inline src/main/java/github/weichware10/toolbox/gui/Tutorial.java:74]:
```suggestion
        //Zuständig für das Laden aller Bilder und Setzten des ersten Bildes
```

File: src/main/java/github/weichware10/toolbox/gui/dialogs/DataBaseDialog.java [hunk 3]
```diff
@@ -65,7 +65,7 @@ public DataBaseClient getDataBaseClient() {
                         controller.getPassword(),
                         controller.getSchema());
             } catch (IllegalArgumentException e) {
-                // display error
+                // Display Fehler
```
Comment (joshuajeschek) [inline src/main/java/github/weichware10/toolbox/gui/dialogs/DataBaseDialog.java:68]:
```suggestion
                // Fehler anzeigen
```

**Review by joshuajeschek — APPROVED** [review]
nais

**Review by ghost — APPROVED** [review]

**Pull request merged**